parser/parser.go

```diff
@@ -152,6 +152,20 @@ func fileDigestMap(path string) (map[string]string, error) {
 	g.SetLimit(max(runtime.GOMAXPROCS(0)-1, 1))
 	for _, f := range files {
 		g.Go(func() error {
+			f, err := filepath.EvalSymlinks(f)
+			if err != nil {
+				return err
+			}
+
+			rel, err := filepath.Rel(path, f)
+			if err != nil {
+				return err
+			}
+
+			if !filepath.IsLocal(rel) {
```

Comment:
should we be using `os.root` here (as in server/create.go)?

Reply:
I'd like to but `os.OpenRoot` errors if `path` is not a directory. Making it compatible for both the directory and file cases requires more refactoring